Client/Application/AppDelegate.swift

```diff
@@ -131,6 +133,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         
         adjustIntegration = AdjustIntegration(profile: profile)
 
+        // We need to check if the app is a clean install to use for
+        // preventing the What's New URL from appearing.
+        if getProfile(application).prefs.stringForKey(LatestAppVersionProfileKey) == nil {
```

Comment:
Is this why you checked for `IntroViewControllerSeenProfileKey` in your first patch? Sorry I asked you to change that, I obviously did not think that through.